package-lock.json

```diff
@@ -10,16 +10,16 @@
       "integrity": "sha512-zbO/DtTnaDappBflIU3zYEgATLToRDmW5uN/EGH1GXaes7ydfjqmAoK++xmJIA+8HfDw7UyPZNdM8fhGhfmMhw==",
```

Comment:
You can remove the `package.json` from the PR by checking out the latest `package.json` from develop and pushing that as a commit to this branch.